x-pack/test/ingest_manager_api_integration/config.ts

```diff
@@ -21,21 +21,25 @@ export default async function ({ readConfigFile }: FtrConfigProviderContext) {
     `${path.join(
       path.dirname(__filename),
       './apis/fixtures/package_registry_config.yml'
-    )}:/registry/config.yml`,
+    )}:/package-registry/config.yml`,
```

Comment:
These changes will go away once this PR is merged: https://github.com/elastic/kibana/pull/70716